### DISCUSSION TITLE
cfclient stuck connecting

### DISCUSSION THREAD

**Discussion by H1ghBre4k3r** [bitcraze/discussions#1606] [Q&A]
Hey everyone!

I followed the [installation instructions](https://github.com/gtfactslab/CrazySim#crazysim-setup) to the point where I start the simulation via

```bash
bash tools/crazyflie-simulation/simulator_files/gazebo/launch/sitl_singleagent.sh -m crazyflie -x 0 -y 0
```

Then, I start `cfclient`, click on `Scan` and the address of the crazyflie pops up (`udp://0.0.0.0:19850`). However, when I click on `Connect`, the client is completely stuck. 

In the logs of the simulation, I can see
```
[Msg] Received CfLib handshake message...
``` 
and the client shows
```
INFO:cflib.crtp.udpdriver:Initialized UDP driver.
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cflib.crazyflie:Callback->Connection initialized[udp://0.0.0.0:19850]
INFO:cflib.crtp.udpdriver:Initialized UDP driver.
INFO:cflib.crazyflie:We are connected[udp://0.0.0.0:19850], request connection setup
INFO:cflib.crazyflie.platformservice:Request _request_protocol_version()
```
After that, nothing happens and I have to kill the client via the terminal. 

My setup is: 
Ubuntu 22.04 (for ARM)
Python 3.10
Gazebo Garden

Thanks a lot for your help!

**Comment by H1ghBre4k3r** [comment]
Closing discussion since it not directly related to this orga. Sorry for the noise. ^^'

**Reply by Bananazooka** [reply]
Thanks!
